workbench-core/accounts/src/services/costCenterService.ts

```diff
@@ -50,7 +51,7 @@ export default class CostCenterService {
       await this._dynamoDbService.updateExecuteAndFormat({
         key: buildDynamoDBPkSk(request.id, resourceTypeToKey.costCenter),
         params: {
-          item: { resourceType: `${this._resourceType}_deleted` }
```

Comment:
You'll want to keep this as well; internally we will still update the resourceType so that this costCenter no longer shows up when listing, but externally we report the status.